templates/lambda-template-python/requirements.txt

```diff
@@ -1 +1 @@
-requests==2.20.0
\ No newline at end of file
+requests
```

Comment:
You may have to change the `deployment-env.yml` files in api/lambda templates to  get  rid of `providerRuntime` key. If we are persisting runtime in service catalog,  we can skip adding this into `deployment-env.yml` file. 

https://github.com/tmobile/jazz/blob/python3/templates/lambda-template-python/deployment-env.yml